components/user/UserPreview.js

```diff
@@ -7,7 +7,7 @@ export default function UserPreview({ profile }) {
       className="flex gap-x-6 rounded-full border-2 border-gray-200 hover:border-gray-500 p-4 my-2"
     >
       <Image
-        src={profile.avatar}
+        src={profile.avatar ? profile.avatar : "https://d1aettbyeyfilo.cloudfront.net/eddiejaoude/19083469_1619470266Fpchubber-hi.webp"}
```

Comment:
Please revert the changes in this file. Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

Reply:
Done that 😀

Reply:
@Panquesito7 hey is my code good to be get merged